### PR TITLE
[Trafodion-1433] [Trafodion-39] [Trafodion -24] SUSE support, pdsh, scanenr

### PR DESCRIPTION
**Pull request by amandamoran**
SUSE Support 
Fixed pdsh errors for SUSE and when systems have banner messages
Scanner not called by default
Trafodion installation on nodes other than all Hadoop nodes
Bug fixes

### CODE REVIEW

File: install/installer/tools/ambari_setup [hunk 1]
```diff
@@ -62,6 +65,9 @@ done
 #Info for log file
 sudo mkdir -p /var/log/trafodion
 sudo chmod 777 /var/log/trafodion
+
+linuxInstalled=$(grep -cr SUSE /proc/version)
```
Comment (DaveBirdsall) [inline install/installer/tools/ambari_setup]: I assume $linuxInstalled is the empty string if it is not SUSE? (Which seems a little odd.) I wonder if a better name for the variable would be $SUSEInstalled?

**Comment by DaveBirdsall** [comment]
Code changes look good to me (though the test failure needs to be explored and addressed). Thanks for taking care of the copyrights and removing the log4cpp license text.

File: install/installer/tools/ambari_setup (hunk 1 above)
Comment (amandamoran) [inline install/installer/tools/ambari_setup]: Actually the grep -c returns a number value. So if SUSE is not installed, it will return a 0 not an empty string. I had to go test this out to make sure if does that (as I couldn't recall) and it does. I did change the variable name to be more clear/.

**Comment by amandamoran** [comment]
jenkins, retest

**Comment by svarnau** [comment]
jenkins, retest -- test scripts updated with new installer config file requirements

**Comment by DaveBirdsall** [comment]
Thanks for the updates. I like the changes. Were you able to discern the causes for the test failures?

File: install/installer/trafodion_install [hunk 2]
```diff
@@ -696,29 +625,30 @@ reservedTrafodionPorts="net.ipv4.ip_local_reserved_ports = ${trafodion_ports}"
...
-      ssh $node -n "sudo echo "$reservedTrafodionPorts","$reserved_ports" >> /etc/sysctl.conf"
+      ssh -q -n $node "sudo echo "$reservedTrafodionPorts","$reserved_ports" >> /etc/sysctl.conf"
    fi
-   ssh $node -n 'sudo chmod 644 /etc/sysctl.conf'
-   ssh $node -n 'sudo /sbin/sysctl -p'
+   ssh -q -n $node 'sudo chmod 644 /etc/sysctl.conf'
+   ssh -q -n $node 'sudo /sbin/sysctl -p' 2>&1 > /dev/null
    else
-      echo "***ERROR: Unable to reserve DCS ports (${trafodion_ports})."
+      echo "***ERROR: Unable to reserve DCS ports (37800 and 40010)."
```
Comment (arvind-narain) [inline install/installer/trafodion_install]: Since the default ports changed (line 623) could you revert this change to utilize the trafodion_ports var ?

Comment (amandamoran) [inline install/installer/trafodion_install]: Thanks for the find!

Fixed!

On Wed, Aug 12, 2015 at 12:29 PM, arvind-narain <notifications@github.com>
wrote:

> In install/installer/trafodion_install
> <https://github.com/apache/incubator-trafodion/pull/50#discussion_r36902901>
> :
>
> >     else
> > -      echo "***ERROR: Unable to reserve DCS ports (${trafodion_ports})."
> > +      echo "***ERROR: Unable to reserve DCS ports (37800 and 40010)."
>
> Since the default ports changed (line 623) could you revert this change to
> utilize the trafodion_ports var ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/50/files#r36902901>.
>



-- 
Thanks,

Amanda Moran

**Comment by amandamoran** [comment]
jenkins, retest

**Comment by DaveBirdsall** [comment]
Tests now pass and comments seem to have been addressed. Seems like it's time to commit this change. I'll commit it now if no-one beats me to it.

**Comment by amandamoran** [comment]
Hooray!!! Thanks Dave!

On Thu, Aug 13, 2015 at 9:18 AM, DaveBirdsall <notifications@github.com>
wrote:

> Tests now pass and comments seem to have been addressed. Seems like it's
> time to commit this change. I'll commit it now if no-one beats me to it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/50#issuecomment-130747392>
> .
>



-- 
Thanks,

Amanda Moran
